### ISSUE TITLE
How do I switch to story mode? Where is the ToC menu? 🤔 

### ISSUE THREAD

**Issue by pdurbin** [ccnmtl/3demos#448]
Hello! I'm in a talk by @nikolas at [LibrePlanet](https://libreplanet.org/2023) and 3Demos seems really interesting but I'm confused by the introduction at https://3demos.ctl.columbia.edu

It says there's a story mode and that I can click on a ToC menu but I'm not sure how to get there. I'll put a screenshot below so you can see what I'm seeing. Thanks!

<img width="1213" alt="Screenshot 2023-03-18 at 11 58 22 AM" src="https://user-images.githubusercontent.com/21006/226117127-428d402b-e330-42a9-ad3a-018fa60642e7.png">

**Comment by pdurbin** [comment]
I just asked @nikolas about this during the Q&A after his talk and it sounds like story mode is still under development. Thanks, Nik. Great talk! 🚀

**Comment by drewyoungren** [comment]
Yeah, it's sort of an unorthodox deplot/develop process as it's being used in class as features are rolled out and tested. Bigger UI redo coming as well. 
Glad to hear it went well.

**Comment by pdurbin** [comment]
@drewyoungren gotcha gotcha, thanks.

@nikolas by the way, this is my list of open source projects at Harvard I mentioned: https://doi.org/10.7910/DVN/TJCLKP

Let's get that webring of open source in higher ed going! 🎉
